packages/eslint-config/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     'plugin:prettier/recommended',
     'plugin:react-hooks/recommended',
   ],
-  plugins: ['prettier', 'tailwindcss', '@dailydotdev/daily-dev-eslint-rules', 'react-hooks'],
```

Comment:
Removed this for now, it gave a JS/TS migration headache and didn't really work last couple weeks anyway